src/momento/incubating/cache_operation_types.py

```diff
@@ -105,10 +105,28 @@ def __init__(self, value: Optional[StoredDictionary], result: CacheGetStatus):
         self._value = value
         self._result = result
 
-    def value(self) -> Optional[StoredDictionary]:
+    def value(self, *, keys_as_bytes: bool = False) -> Optional[StoredDictionary]:
+        """Get the dictionary as stored in the cache.
+
+        By default unmarshals the keys to strings. To override this
+        set `keys_as_bytes` to `True`.
+
+        Values are of type `CacheDictionaryValue`, which implement `value` and
+        `value_as_bytes` methods to unmarshal to string or bytes respectively.
+
+        Args:
+            keys_as_bytes (bool, optional): Leave the keys as uninterpreted bytes. Defaults to False.
+
+        Returns:
+            Optional[StoredDictionary]: The dictionary if the cache get was a hit, else None.
+        """
         if self.status() != CacheGetStatus.HIT:
             return None
-        return self._value
+
+        if keys_as_bytes:
+            return self._value
+
+        return {cast(bytes, k).decode("utf-8"): v for k, v in self._value.items()}  # type: ignore
```

Comment:
style nit but maybe format this like you did in [src/momento/incubating/aio/utils.py](https://github.com/momentohq/client-sdk-python/pull/104/files#diff-d7daa9b9f7bcb6c91556a651e032307389e6b2e0cd0edede623a4182e288ae49) I thought made easier to read

Reply:
The black code formatter is really strict and will override it. Would prefer a separate PR for tweaking that.